### ISSUE TITLE
Donkey Kong 64 camera floating with lr-mupen64plus-nx

### ISSUE THREAD

**Issue by retropieuser** [gonetz/GLideN64#2392]
I've found that with DK64 (jpn, pal and us versions) the camera seems to not work until I change the .ini file to FrameBufferEmulation\copyDepthToRDRAM=0. Can we change this please or will it break other stuff in doing so?

**Comment by gonetz** [comment]
Hmm, I can't run DK64 with mupen64plus-gui. It hangs after the start logo.
I'm not sure that copyDepthToRDRAM is necessary for anything in this game.
I suppose we can disable it in custom settings for this game.

**Comment by retropieuser** [comment]
Yeah I should point out I was using lr-mupen64plus-next on RetroPie (on a RPI4 2gb). This is a video of the error of I was getting. 

https://www.youtube.com/watch?v=P5wDA_2_gWI

Once we changed the build so in the GlideN64.ini to turn off copyDepthToRDRAM, it fixed the camera here.

**Comment by standard-two-simplex** [comment]
>  I'm not sure that copyDepthToRDRAM is necessary for anything in this game.

There was the issue of photographing the fairy. #2026

**Comment by Searinox** [comment]
"Copy video card depth buffer to N64 memory" = "From VRAM" is needed for issue #2026. However, the camera falsely detects an obstruction and attempts to reset its position every second only if this is used in conjunction with "N64-style depth compare" as anything other than "Disabled".

Tested with
GlideN64 commit 93524b65f7ce68438f400abb6b4f0b5195984725
on Project64 2.4.0.1416-g88ac97c x32 with default RSP, graphics HLE mode

**Comment by gonetz** [comment]
So "N64-style depth compare" should be set to "Disabled".

**Comment by retropieuser** [comment]
Thanks, that makes sense! I didn't do enough digging to find the old issue from last year. It makes sense though as I think the file got edited and before that the camera bug wasn't there.

[DONKEY%20KONG%2064]
Good_Name=Donkey Kong 64 (E)(J)(U)
frameBufferEmulation\copyDepthToRDRAM=0
frameBufferEmulation\N64DepthCompare=0

I just tested it on the mupen64plus on the RPI4 and it works, I also tested the save state from that previous issue and was able to photograph the fairy ok.

**Comment by fzurita** [comment]
Interesting, seeing N64 depth compare to disabled in Android didn't seem to fix the issue there. Only using software depth render does.

**Comment by retropieuser** [comment]
Oh really? I can't even get lr-mupen64plus-next to run on my android smartphone (although I have an old phone, Galaxy A5) any game I select crashes retroarch.

**Comment by fzurita** [comment]
Try "m64plus fz" in the play store and set the emulation profile to GLideN64-Medium.

**Comment by retropieuser** [comment]
Ok I gave that a download and I see what you mean, do you know what directory m64plus saves save states into? As I'd be curious to see if it's possible to photograph the fairy or if it becomes an issue again.

**Comment by fzurita** [comment]
You have to go to Tools then "Import/Export data". From there "Export game data".

**Comment by retropieuser** [comment]
Ok I got the save slot working, thanks! 

And I definitely can't photograph the banana fairy with that emulator using software or hardware depth render. Unless I just suck at using android touchscreen controls

**Comment by fzurita** [comment]
Interesting, try switching to the "Software renderer" emulation profile. It will be slow, but it should let you photograph it.

**Comment by retropieuser** [comment]
It does let you photograph it doing that, but using software renderer is using the angrylion plugin I think? So it'd probably make more sense on that emulator to play the whole game on that plugin
